tests/test_packaging.py

```diff
@@ -76,6 +76,7 @@ def test_requirements_files():
         | req_set_dict["requirements-dev-contrib.txt"]
         | req_set_dict["requirements-dev-lite.txt"]
         | req_set_dict["requirements-dev-api-docs-test.txt"]
+        | req_set_dict["requirements-dev-snowflake.txt"]
```

Comment:
The only reason I think think of why Databricks would be failing is this new requirement. I don't think we should be changing this here.